docs/features.md

```diff
@@ -0,0 +1,315 @@
+  For the view to be shown, you must enable the feature. Moreover, all the CRDs defined in the selectors array must exist in a cluster.
+
+  Default settings:
+
+  ```bash
+  "APPLICATIONS":{
+    "isEnabled": true,
+    "selectors":[
+      {
+        "type":"apiGroup",
+        "apiGroup":"applicationconnector.kyma-project.io"
+      }
+    ]
+  },
+  ```
+
+- **BTP_CATALOG** – is used to show or hide the BTP Catalog view and to define which CRDs are required for the view to be shown properly.  
+  For the view to be shown, you must enable the feature. Moreover, all the CRDs defined in the selectors array must exist in a cluster.
+
+  Default settings:
+
+  ```bash
+  "BTP_CATALOG":{
+    "isEnabled": true,
+    "selectors":[
+      {
+        "type":"apiGroup",
+        "apiGroup":"services.cloud.sap.com"
+      }
+    ]
+  },
+  ```
+
+- **CUSTOM_DOMAINS** – is responsible for displaying views of DNS Entry, DNS Provider, Gateway, Issuer and Certificate. For the view to be shown, you must enable the feature. Moreover, all the CRDs defined in the selectors array must exist in a cluster.
+
+  Default settings:
+
+  ```bash
+  "CUSTOM_DOMAINS": {
+    "isEnabled": true,
+    "selectors": [
+      {
+        "type": "apiGroup",
+        "apiGroup": "dns.gardener.cloud"
+      }
+    ]
+  },
+  ```
+
+- **EVENTING** – is used to show or hide the Eventing view and to define which CRDs are required for the view to be shown properly.  
+  It is also used to determine if the Event Subscriptions should be displayed in Function and Service details.  
+  For the view to be shown, you must enable the feature. Moreover, all the CRDs defined in the selectors array must exist in a cluster.
+
+  Default settings:
+
+  ```bash
+  "EVENTING ":{
+    "isEnabled": true,
+    "selectors":[
+      {
+        "type":"apiGroup",
+        "apiGroup":"eventing.kyma-project.io"
+      }
+    ]
+  },
+  ```
+
+- **KUBECONFIG_ID** – is used to configure the URL to which Busola sends a request to download a kubeconfig file. If you add `?kubeconfigID={your ID}` to the Busola URL, Busola tries to download the kubeconfig from the {kubeconfigUrl}/{yourID} . If the operation succeeds, Busola adds the kubeconfing file to the cluster.
+  If you use a full address in the **kubeconfigUrl** field, Busola also reads it.
+
+  Default settings:
+
+  ```bash
+  "KUBECONFIG_ID": {
+   "isEnabled": true,
+   "config": {
+     "kubeconfigUrl": "https://kyma-env-broker.cp.dev.kyma.cloud.sap/kubeconfig"
+   }
+  },
+  ```
+
+- **LEGAL_LINKS** – is used to show or hide legal links. You can find the all available links in the below example.  
+  In **config** you can find the unchangeable keys (you cannot use **legalDisclosure** instead of **legal-disclosure**). The keys include the default link, which takes you to the default address, and a link that depends on your chosen language.
```

Comment:
```suggestion
  In **config** you can find the unchangeable keys (you cannot use **legalDisclosure** instead of **legal-disclosure**). The keys include both the default link, which takes you to the default address, and a link that depends on your chosen language.
```